cmd/ossyscall.go

```diff
@@ -20,8 +20,17 @@ import (
 	"os"
 	"os/signal"
 	"syscall"
+
+	"github.com/dapr/cli/pkg/print"
 )
 
 func setupShutdownNotify(sigCh chan os.Signal) {
 	signal.Notify(sigCh, syscall.SIGTERM, syscall.SIGINT)
 }
+
+// createProcessGroupID creates a process group id for the current process.
+func createProcessGroupID() {
+	if err := syscall.Setpgid(0, 0); err != nil {
```

Comment:
Can point to the manpage and explain the significance of `0, 0` params.

Reply:
@pravinpushkar can you please add this?

Reply:
added